src/parsers/index.js

```diff
@@ -0,0 +1,20 @@
+import cucumberParser from './cucumber'
+import multiTestParser from './multitest'
+import standardParser from './standard'
+
+let all = [cucumberParser, multiTestParser, standardParser]
+
+function getParser (name, output = '') {
+  let parser
+  if (name) {
+    parser = all.find((parser) => name === parser.name)
+    if (!parser) { throw new Error(`Invalid Parser Specified: ${name}`) }
+  } else {
+    parser = all.find((p) => {
+      return p.test(output)
+    })
```

Comment:
You mean creating a new header for the parsers?
I was thinking more along the lines of adding the parser option in the usage header:

Hmm that kinda sounds like users have to write their own parser module XD

How about

⭐Parsers⭐
The following parsers can be used to discover and rerun failed tests.
- StandardParser: will parse stack trace error messages and rerun the testfiles that resulted in these error.
- CucumberParser: will parse cucumber test output and rerun the failed tests.
- multiTestParser: will parse multi-capabilities or sharded test file output and rerun the failed tests.

And then also adding the parser option to the 'usage' header.